app/src/main/res/values/strings.xml

```diff
@@ -57,6 +57,8 @@
     <string name="background_settings_title">Run in background</string>
     <string name="boot_settings_title">Start automatically</string>
     <string name="stop_svc_when_leaving">Stop service after leaving app</string>
+    <string name="stop_svc_when_leaving_summary_on">The service will be stopped automatically when the application is cleared from recents, and also after a timeout if the application is running in the background but no transfers are taking place</string>
```

Comment:
Maybe I'd make it a little shorter so that it doesn't take 5 lines. Otherwise it looks good.
```suggestion
    <string name="stop_svc_when_leaving_summary_on">The service will be stopped automatically when cleared from recents or after a period of inactivity</string>
```

Reply:
I agree, especially since I'm not very good at technical terms in English (it was more of a test version of the description). My native language is Russian and if necessary I will try to shorten the lines in the translation as well.